domains/wissembagga.is-a.dev.json

```
@@ -0,0 +1,9 @@
+{
+  "owner": {
+    "email": "wissbagga@gmail.com"
+  },
+  "record": {
+    "cname": "wissembagga.vercel.app",
+    "txt": "vc-domain-verify=wissembagga.is-a.dev,b16a1fded1354394c796"
```

TXT verification should be on a nested subdomain, please read the [docs](https://docs.is-a.dev/guides/vercel/)
```suggestion
    "CNAME": "wissembagga.vercel.app"
```